.travis.yml

```diff
@@ -7,49 +7,39 @@ cache:
     - $HOME/.composer/cache
 
 php:
- - 5.5
-# - 5.6
  - 7.1
+ - 7.2
+
+services:
+ - mysql
+ - postgresql
 
 env:
  global:
-  - IGNORE_PATHS=sdk/,fonts/,pix/
-  - IGNORE_NAMES=jquery*,moment.js,colorbox.css,tooltipster.css,*.min.css,*.min.js,tii-icon-webfont.css,styles.css,styles.css.map,gulpfile.js
+  - IGNORE_PATHS=sdk/,pix/,vendor/
+  - IGNORE_NAMES=jquery*,styles.css,autoload_static.php
```

Comment:
Switch these back to what they were. We need to ignore all of those

Reply:
Done :D